extensions/docs/msal-node-extensions.md

```diff
@@ -109,9 +166,61 @@ const publicClientConfig = {
 const pca = new PublicClientApplication(publicClientConfig);
 ```
 
+Example (for Electron node-js desktop app):-
+
+authConfig.js:-
+```js
+const AAD_ENDPOINT_HOST = "https://login.microsoftonline.com/"; // include the trailing slash
+const REDIRECT_URI = "ENTER_REDIRECT_URI";
+
+const cachePath = "path/to/cache/file.json";
+
+/*define persistence config based on the appropriate persistence you are using(e.g- FilePersistenceWithDataProtection, generic PersistenceCreateor, etc)*/
+
+//defining persistence config for PersistenceCreator
+const persistenceConfiguration = {
+    cachePath,
+    dataProtectionScope: DataProtectionScope.CurrentUser,
+    serviceName: "test-msal-electron-service",
+    accountName: "test-msal-electron-account",
+    usePlaintextFileOnLinux: false,
+}
+
+  const msalConfig = {
+    auth: {
+        clientId: "CLIENT_ID_HERE",
+        authority: `${AAD_ENDPOINT_HOST}TENANT_ID_HERE`,
+    },
+    cache: {
+        cachePlugin: null // set later in main.js as shown above 
+    },
+    system: {
+        loggerOptions: {
+            loggerCallback(loglevel, message, containsPii) {
+                console.log(message);
+            },
+            piiLoggingEnabled: false,
+            logLevel: LogLevel.Verbose,
+        },
+    },
+};
+...
+
+module.exports = {
+  msalConfig: msalConfig,
+  protectedResources: protectedResources,
+  REDIRECT_URI: REDIRECT_URI,
+  persistenceConfiguration
+};
+
+```
+
 Note that MSAL will not read and write to persistence by default. You will have to call PublicClientApplication.tokenCache.readFromPersistence() and PublicClientApplication.tokenCache.writeToPersistence() anytime you trigger and MSAL Node operation that alters the token cache.
 
+
 ### Note for Electron Developers:
+Electron sample  link will be added here soon for clarity. 
+
```

Comment:
@derisen Thanks for your help in going over the documentation gaps and creating a separate work item to add the sample here ! Appreciate further feedback. 
CC: @salman90 @sameerag

Reply:
Let's place [this link](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/extensions/samples/electron-webpack) now, since we will eventually update it.

Reply:
Oh there's [this sample](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/extensions/samples/msal-node-extensions) too -let's link both of them in this doc

Reply:
This https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/extensions/samples/electron-webpack is fairly self-sufficient, the second link seems related to this one. Should I add both here? @derisen

Reply:
Wouldn't hurt I think. I like to end the docs with a #samples section listing the related samples, but up to you.